docs/content/Examples-Tutorials-Recipes/Using-different-schemas-for-tenants.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Using Different Schemas for Tenants
+permalink: /recipes/using-different-schemas-for-tenants
+category: Examples & Tutorials
+subCategory: Access control
+menuOrder: 2
+---
+
+## Use case
+
+We want to manage user access to different Cubes. In the recipe below, we'll
+learn how to use multiple data schemas for various tenants.
+
+## Configuration
+
+We have two tenants and we created folders with the data schema for each one
+inside the `schema` folder. The folders are named such as a tenants. Then we
```

Comment:
```suggestion
We have a folder structure as follows:

```bash
schema
├── avocado
│   ├── LineItems.js
│   ├── Orders.js
│   ├── ProductCategories.js
│   ├── Products.js
│   ├── Suppliers.js
│   └── Users.js
└── mango
    ├── LineItems.js
    ├── Orders.js
    ├── ProductCategories.js
    ├── Products.js
    ├── Suppliers.js
    └── Users.js
```

The above suggestion isn't completely correct because GitHub won't let me add codefences in suggestions properly

Reply:
@hassankhan great idea! I've added the correct structure for this recipe. Thank you!